{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Product defects","children":[{"name":"The following error originated from your application code, not from Cypress.\n\n  > Additional keys were returned from `getServerSideProps`. Properties intended for your component must be nested under the `props` key, e.g.:\n\n\treturn { props: { title: 'My Title', content: '...' } }\n\nKeys that need to be moved: userId, id, title, completed.\nRead more: https://nextjs.org/docs/messages/invalid-getserversideprops-value\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Check that Shop navigation link works","uid":"dd4fde4e8bae28e3","parentUid":"9923ecdd3da820d4b0811512ad94f2f1","status":"failed","time":{"start":1699353811501,"stop":1699353813836,"duration":2335},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":true,"parameters":[],"tags":[]},{"name":"Check that Shop text navigation link works","uid":"df38b18be01f10a3","parentUid":"9923ecdd3da820d4b0811512ad94f2f1","status":"failed","time":{"start":1699353830117,"stop":1699353832653,"duration":2536},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":3,"retriesStatusChange":true,"parameters":[],"tags":[]}],"uid":"9923ecdd3da820d4b0811512ad94f2f1"},{"name":"The following error originated from your application code, not from Cypress.\n\n  > dispatcher.useSyncExternalStore is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Checks increase/decrease actions by React buttons + check counter can have negative value","uid":"e7901f805ba3b73c","parentUid":"8293d5267302889c2a48aca72e30d861","status":"failed","time":{"start":1699354232629,"stop":1699354233330,"duration":701},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"Checks increase/decrease actions by Vue buttons + check counter can have negative value","uid":"7e1ac64712c3ca81","parentUid":"8293d5267302889c2a48aca72e30d861","status":"failed","time":{"start":1699354233882,"stop":1699354234386,"duration":504},"flaky":false,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":1,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"8293d5267302889c2a48aca72e30d861"},{"name":"The following error originated from your application code, not from Cypress.\n\n  > Module \"./NoLongerWorks\" does not exist in container.\nwhile loading \"./NoLongerWorks\" from webpack/container/reference/app2\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","children":[{"name":"Check App 1 app name visibility","uid":"5a3b579dca916e85","parentUid":"fa9c2129dc7eff94d6619130208a38f1","status":"failed","time":{"start":1699354610388,"stop":1699354611146,"duration":758},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"Check App 1 contains button","uid":"953748071430475c","parentUid":"fa9c2129dc7eff94d6619130208a38f1","status":"failed","time":{"start":1699354611894,"stop":1699354612539,"duration":645},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"Checks that button in App 1 is not disabled","uid":"2dd3846e472af603","parentUid":"fa9c2129dc7eff94d6619130208a38f1","status":"failed","time":{"start":1699354613179,"stop":1699354613896,"duration":717},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]},{"name":"Checks that apps names is not equal","uid":"9ce37174e5b35aaf","parentUid":"fa9c2129dc7eff94d6619130208a38f1","status":"failed","time":{"start":1699354624200,"stop":1699354625520,"duration":1320},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"fa9c2129dc7eff94d6619130208a38f1"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"},{"name":"Test defects","children":[{"name":"`cy.visit()` failed trying to load:\n\nhttp://localhost:3002//shop\n\nThe response we received from your web server was:\n\n  > 500: Internal Server Error\n\nThis was considered a failure because the status code was not `2xx`.\n\nThis http request was redirected '1' time to:\n\n  - 308: http://localhost:3002/shop\n\nIf you do not want status codes to cause failures pass the option: `failOnStatusCode: false`","children":[{"name":"Check the header content of Shop page","uid":"a6aa75a48f605ed3","parentUid":"fc7d2bf7fe16529721b0bcd46f598781","status":"broken","time":{"start":1699353816790,"stop":1699353817188,"duration":398},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":2,"retriesStatusChange":true,"parameters":[],"tags":[]}],"uid":"fc7d2bf7fe16529721b0bcd46f598781"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}